src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/DateTimePicker/DateTimePicker.jsx

```diff
@@ -12,9 +12,15 @@ const variantMapper = {
 };
 
 export const DateTimePicker = ({id, field, editorContext}) => {
-    const displayDateFormat = field.formDefinition.selectorOptions.find(option => option.name === 'format');
     const variant = variantMapper[field.formDefinition.selectorType];
-    const disabledDays = fillDisabledDaysFromJCRConstraints(field, variant === 'datetime');
+    const isDateTime = variant === 'datetime';
+    const disabledDays = fillDisabledDaysFromJCRConstraints(field, isDateTime);
+
+    // Handle the date format, only "en" have a specific format.
+    let displayDateFormat = editorContext.uiLang === 'en' ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
Question: we use the language of the browser, don't we?

Reply:
no we use the preferred languages:
it's clearly mentioned here: https://jira.jahia.org/browse/BACKLOG-10497?focusedCommentId=96029&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-96029

Reply:
What did you mean by "preferred languages" of: 
- Browser?
- Content editor UI?